### DISCUSSION TITLE
privategpt相关问题

### DISCUSSION THREAD

**Discussion by BondChang** [ymcui/Chinese-LLaMA-Alpaca#584] [Q&A]
https://github.com/ymcui/Chinese-LLaMA-Alpaca/wiki/%E4%BD%BF%E7%94%A8privateGPT%E8%BF%9B%E8%A1%8C%E5%A4%9A%E6%96%87%E6%A1%A3%E9%97%AE%E7%AD%94

![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/32505512/dba22f17-3938-4736-91f7-fb883fac1842)

请问文档中的modelpath是自己合并好的模型吗

**Comment by ymcui** [comment]
里面已经写了，model_path指的是llama.cpp支持的ggml文件或者其他privateGPT支持的格式。

P.S. 麻烦不要重复开贴，谢谢。
